### DISCUSSION TITLE
Is it possible to show a sponsors list on readme?

### DISCUSSION THREAD

**Discussion by laike9m** [community/community#22753] [New to GitHub]
<p>opencollective has this feature, so that you can show all the backers in readme:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/d/ad0ee768291c7612ef83a7afea95506a1a354d76.png" data-download-href="/uploads/short-url/oGWx6noII3M0vKJNrAeWq48N2Hc.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/2592205/181073018-3cc6c9c3-e42e-4451-8d26-643ce22d6c77.png" alt="image" data-base62-sha1="oGWx6noII3M0vKJNrAeWq48N2Hc" width="690" height="377" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/d/ad0ee768291c7612ef83a7afea95506a1a354d76_2_690x377.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/d/ad0ee768291c7612ef83a7afea95506a1a354d76_2_1035x565.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/a/d/ad0ee768291c7612ef83a7afea95506a1a354d76.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/d/ad0ee768291c7612ef83a7afea95506a1a354d76_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1278×700 170 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lee-dohm** [comment, marked as answer]
<p>There’s nothing built in that makes that happen, no. You could create some automation that did this using GitHub Actions or some other framework to update the README every time the sponsors list changes.</p>
<p>On the other hand, a sponsors list is shown on the user’s or organization’s profile page automatically. The benefit of this is that it respects the sponsoring user’s selection for whether they want their sponsorship to be public or private.</p>
<p>Let us know if you have more questions!</p>

**Comment by laike9m** [comment]
<p>Thanks. I do hope it can be added some day :slightly_smiling_face:</p>

**Comment by jessuppi** [comment]
This GitHub Action is a popular solution:

https://github.com/JamesIves/github-sponsors-readme-action
